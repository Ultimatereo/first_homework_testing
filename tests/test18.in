5
4 2 2 0 0 4
5 2 1 2 3 1
2 4 2 3 0 4
0 1 3 3 4 2
1 5 3 1 2 5
