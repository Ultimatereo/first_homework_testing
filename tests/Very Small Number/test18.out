0.130435
0.391304
1.34783
-0.173913
-0.478261
